icons/meson.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><title>Meson</title><desc>Meson Build System</desc><path d="M8.06418846,0 C3.10639742,2.82936169 0.0282180293,8.13062146 0,13.8888618 C5.18688415,16.7473408 11.2775825,16.6498493 15.9975608,13.9189583 C15.9981506,13.9051605 15.9988679,13.8912515 16,13.8775118 C15.9940495,8.1501713 12.9691622,2.86051562 8.06418846,0 Z M10.7332152,6.13083427 C11.4682585,6.6357224 11.3592575,7.85129556 11.2408548,8.68362229 C11.0901929,9.60047886 10.2205397,11.1078066 9.09069893,12.0914493 C9.23684988,12.3032615 9.3223187,12.5640986 9.32108911,12.8423862 C8.62095159,13.2001085 7.92621976,12.7965367 7.08599926,12.0692471 C6.24578126,11.3419498 5.33957041,9.88481411 5.03883634,8.38709586 C4.77209337,8.41564635 4.49895504,8.35539275 4.26468117,8.20602183 C4.32587063,7.36620508 5.20768967,6.86737743 5.98919987,6.57290981 C6.77071258,6.27844144 8.78406185,6.17877039 10.2036054,6.68885479 C10.3142933,6.45395139 10.4976467,6.25645792 10.7332152,6.13132763 L10.7332152,6.13083427 Z" id="path817-6-2-9-7-9-93-9-8-1" fill="#3f51b5"></path></svg>
```

Comment:
Looks really good so far, thanks for changing it. As a last task, could you please make the icon a little bit smaller, so that there's some transparent space around it and position it in the middle.

Example, if the yellow circle would be your icon, then there's some space around it:

<img src="https://user-images.githubusercontent.com/12248527/78368070-5a2a1000-75c3-11ea-8058-614787f11e6d.png" alt="circle example space around" width="230px" />

Reply:
![Screenshot from 2020-04-03 21-55-58](https://user-images.githubusercontent.com/62474785/78374362-f5d47400-75f5-11ea-9b54-005383289518.png)
Hope this margin is okay ?